tools/c7n_left/c7n_left/cli.py

```diff
@@ -75,6 +79,7 @@ def test(policy_dir, filters):
         policy_dir=policy_dir,
         output_file=sys.stdout,
         filters=filters,
+        var_file=None,
```

Comment:
Wouldn't `Config.empty` default the `var_file` to None?

Reply:
no.. its custodian config not left customized, config is a bag / dict w/ attr access, accessing an unknown key still raises.

Reply:
but this value should likely be an empty string.